rotkehlchen/accounting/history_base_entries.py

```diff
@@ -0,0 +1,269 @@
+import logging
+from collections.abc import Iterator
+from typing import TYPE_CHECKING, Any, TypeVar
+from rotkehlchen.accounting.ledger_actions import LedgerActionType
+
+from rotkehlchen.accounting.mixins.event import AccountingEventMixin, AccountingEventType
+from rotkehlchen.accounting.structures.base import HistoryBaseEntry
+from rotkehlchen.accounting.structures.evm_event import EvmEvent
+from rotkehlchen.accounting.structures.types import HistoryEventSubType, HistoryEventType
+from rotkehlchen.chain.evm.accounting.structures import (
+    BaseEventSettins,
+    TxAccountingTreatment,
+    TxEventSettings,
+)
+from rotkehlchen.chain.evm.decoding.constants import CPT_GAS
+from rotkehlchen.logging import RotkehlchenLogsAdapter
+from rotkehlchen.types import Timestamp
+
+if TYPE_CHECKING:
+    from rotkehlchen.accounting.pot import AccountingPot
+    from rotkehlchen.chain.evm.accounting.aggregator import EVMAccountingAggregators
+
+logger = logging.getLogger(__name__)
+log = RotkehlchenLogsAdapter(logger)
+
+
+def make_default_accounting_settings(pot: 'AccountingPot') -> dict[str, BaseEventSettins]:
+    """
+    Returns accounting settings for events that can come from various decoders and thus don't have
+    any particular protocol. These settings also allow users to customize events in the UI.
+    Users are supposed to apply these settings in the history view.
```

Comment:
apply these settings in the history view? What is this supposed to mean? This is moved from the old docstring so perhaps it's speaking in the future tense. I think it refers to customization of how they should be handled in accounting, somethign which is not yet possible. The issue is here: https://github.com/rotki/rotki/issues/4341

So perhaps write:

```suggestion
    Users are supposed to apply these settings in the history view once they become customizable: https://github.com/rotki/rotki/issues/4341
```